sos/report/plugins/mysql.py

```diff
@@ -36,6 +36,7 @@ def setup(self):
             "/var/log/mysql/mysqld.log",
             "/var/log/mariadb/mariadb.log",
             "/var/lib/mysql/grastate.dat"
+            "/var/lib/mysql/gvwstate.dat"
```

Comment:
Missing a comma at the end of line 38, hence the implicit concatenation error from LGTM.